src/main/java/org/javarosa/core/model/SelectChoice.java

```diff
@@ -75,6 +81,16 @@ public String getValue() {
         return value;
     }
 
+    public String getChild(String childName) {
```

Comment:
I also considered `getProperty`. I went with `getChild` because it best matches the underlying tree representation. `Property` makes the most sense for `geojson` and `column` makes the most sense for CSV.